templates/datasets.html

```diff
@@ -142,76 +148,55 @@ <h4 class="mb-3">Upload Datasets</h4>
         </div>
 
         <!-- Public Subgroup -->
-        <div id="public">
-            <!-- Include public divs here -->
-            <!-- Portfolio item 8-->
-            {% for file in datasets.public %}
-            <div class="portfolio-item" id="portfolioItem{{ forloop.counter }}">
-                <a class="portfolio-link" data-bs-toggle="modal" href="#portfolioModal{{ forloop.counter }}">
-                    <img class="img-fluid" src="/static/assets/img/d2.jpg" alt="..." />
-                </a>
-                <div class="portfolio-caption">
-                    <div class="portfolio-caption-heading">{{ file }}</div>
-                    <div class="portfolio-caption-subheading text-muted">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Integer nec odio. Praesent libero. Sed cursus ante dapibus diam. Sed nisi.</div>
-                    <!-- Buttons -->
-                    <div class="d-flex align-items-center mt-2">
+        <div class="text-center">
+            <h2 class="section-heading text-uppercase">Datasets</h2>
+        </div>
+        <table class="table table-striped">
```

Comment:
Only public subgroup is formatted as table and not for private/favourites subgroup.

Reply:
Yes, me and professor worked on changing the models with correct name and adding the private key and foreign key. I pulled up from your existing branch. It is related to frontend and I believe you have to take care of it.

Reply:
Anyways, I myself made the changes in the html and in wrote appropriate code in views.py too. Now table is added. This issue is resolved